classes/Currency.php

```diff
@@ -0,0 +1,128 @@
+<?php
+class Currency {
```

Comment:
This is cool! I almost wonder if we should take this a step further and make it a `UserCurrency` class that has fully named variables for every type of user currency, calls would look like:

`$player->money->amount` => `$player->currency->money`
`$player->money->add()` => `$player->currency->addMoney()`

I'm not sure which way is better, okay with your approach as is but something to ponder

Reply:
I like the `$player->currency->addMoney()` methodology